### PR TITLE
Handle draft stages for IWA

### LINKED ISSUE #12: Handle IWA identifiers (and its draft stages)
We need to handle the IWA draft stages.

```
AWI IWA 36
AWI IWA 39
AWI IWA 41
AWI IWA 42
CD IWA 37
CD IWA 37-1
CD IWA 37-2
CD IWA 37-3
CD IWA 40
CD IWA 42
ISO/IWA 32:2019(en)
ISO/WD IWA 19
IWA 14-1:2013
IWA 19:2017
IWA 2 N3
IWA 24:2016
IWA 26:2017
IWA 31:2020
IWA 32:2019
IWA 34:2021
IWA 34:2021(en)
IWA 35:2020
IWA 37-1:2022
IWA 37-2:2022
IWA 37-3:2022
IWA 40:2022(en)
IWA 42
IWA 8:2009
NP IWA 30-2
NP IWA 32
NP IWA 33
PRF IWA 36
PRF IWA 37-1
PRF IWA 37-3
PRF IWA 40
PRF IWA 42
WD IWA 19
```

We need to parse items like:
```
ISO/IWA 32:2019(en)
ISO/WD IWA 19
```

But the correct forms are:
```
IWA 32:2019(en)
WD IWA 19
```

(Updated list)

### PR DESCRIPTION
**Pull request by mico**
closes #12

### CODE REVIEW

File: lib/pubid/iso/transformer.rb [hunk 1]
```diff
@@ -44,6 +44,8 @@ def self.convert_stage(code)
         "FDIS"
       when "Fpr"
         "PRF"
+      when "AWI"
+        "NP"
```
Comment (ronaldtse) [inline lib/pubid/iso/transformer.rb:48]: As mentioned https://github.com/metanorma/pubid-iso/issues/12#issuecomment-1062069154 , "NP" != "AWI".

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: lib/pubid/iso/parser.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@ class Parser < Parslet::Parser
 
     rule(:stage) do
       str("NP") | str("WD") | str("CD") | str("DIS") | str("FDIS") | str("PRF") |
-        str("IS") | str("AWI") | str("FD") | str("D")
+        str("IS") | str("AWI") | str("FD") | str("D") | str("AWI")
```
Comment (ronaldtse) [inline lib/pubid/iso/parser.rb:26]: Wait, why are there two instances of "AWI"?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: lib/pubid/iso/transformer.rb [hunk 3]
```diff
@@ -56,6 +56,8 @@ def self.convert_stage(code)
         "PRF"
       when "pD", "PD"
         "CD"
+      when "AWI"
+        "WD"
```
Comment (mico) [inline lib/pubid/iso/transformer.rb:60]: @ronaldtse Is that ok? I know WD != AWI, but it has the same stage number "20.00"

Comment (ronaldtse) [reply lib/pubid/iso/transformer.rb:60]: ## AWI is stage 20.00

Screenshot 2022-03-11 at 12 09 55 AM

## WD is stage 20.20 or 20.60 (i.e. 20.20+)

Screenshot 2022-03-11 at 12 09 10 AM

Screenshot 2022-03-11 at 12 08 54 AM

## FDIS is 50.00+, but PRF can also be 50.00+

PRF and FDIS cannot really be distinguished.

PRFs being 50.00, 50.20:
Screenshot 2022-03-11 at 12 16 49 AM
Screenshot 2022-03-11 at 12 17 31 AM
Screenshot 2022-03-11 at 12 18 13 AM

FDIS being 50.00, 50.20:
Screenshot 2022-03-11 at 12 16 58 AM
Screenshot 2022-03-11 at 12 18 58 AM

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**